app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
```

Comment:
The `people` dictionary should be accessed as a class attribute using `Person.people` instead of `self.people`. This ensures that the dictionary is shared across all instances of the `Person` class.